### PR TITLE
Finish Intro-Python-I

### PR DESCRIPTION
**Pull request by andrewlowell**
Done-zo :)

### CODE REVIEW

File: src/03_modules.py [hunk 1]
```diff
@@ -11,21 +11,34 @@
...
 
+print(sys.implementation)
 
 import os
 # See the docs for the OS module: https://docs.python.org/3.7/library/os.html
 
 # Print the current process ID
 # YOUR CODE HERE
 
+print(os.getegid())
```
Comment (Skidragon) [inline src/03_modules.py]: I think you meant, `os.getpid()`

File: src/04_printing.py [hunk 2]
```diff
@@ -12,6 +12,12 @@
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
 
+print("x is " + str(x) + ", y is " + str(y) + ", z is \"" + z + "\"")
```
Comment (Skidragon) [inline src/04_printing.py]: You can also do:
`print("x is %d, y is %.2f, z is \"%s\"" % (x, y, z))`

File: src/10_functions.py [hunk 3]
```diff
@@ -2,6 +2,12 @@
 
 # YOUR CODE HERE
 
+def is_even(number):
+  if number % 2 == 0:
+    return True
+  else:
+    return False
+
```
Comment (Skidragon) [inline src/10_functions.py:7]: Can be simplified to this:
```
def is_even(n):
  return n % 2 == 0
```

File: src/13_file_io.py [hunk 4]
```diff
@@ -8,11 +8,21 @@
 # Open up the "foo.txt" file (which already exists) for reading
 # Print all the contents of the file, then close the file
 
+f = open('foo.txt')
+read_data = f.read()
+print(read_data)
+f.close()
+
```
Comment (Skidragon) [inline src/13_file_io.py:15]: This way is good also, here is just another way to do it.
```
with open("foo.txt") as fp:
  for line in fp:
    print(line)

fp.close()
```

File: src/13_file_io.py [hunk 5]
```diff
@@ -8,11 +8,21 @@
...
 # sure that it contains what you expect it to contain
 
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
+
+f = open("bar.txt", "w")
+f.write("This is line one, yo\n")
+f.write("This is line two, yo\nThis is line three, yo\n")
+f.close()
```
Comment (Skidragon) [inline src/13_file_io.py:29]: Can be simplified to this:

```
fp = open("bar.txt", "w")

fp.write("""Line 1
            Line 2
            Line 3""")

fp.close()
```

Comment (andrewlowell) [reply src/13_file_io.py:29]: This seems useful, though it does pick up any indentation :joy:

File: src/03_modules.py (hunk 1 above)
Comment (andrewlowell) [reply src/03_modules.py]: I absolutely did.

File: src/04_printing.py (hunk 2 above)
Comment (andrewlowell) [reply src/04_printing.py]: I tried it, not having connected the strings, glad to know this works.

File: src/10_functions.py (hunk 3 above)
Comment (andrewlowell) [reply src/10_functions.py:7]: Good call.

**Pull request merged**